lstmcpipe/stages/mc_dl2_to_irfs.py

```diff
@@ -154,7 +110,7 @@ def main(dl2_directory, config_file, irf_point_like=True, irf_gamma_offset='off0
     log_dl2_to_irfs = {}
     list_job_id_dl2_irfs = []
 
-    if not flag_full_workflow or log_from_dl1_dl2 == {}:
+    if log_from_dl1_dl2 == {}:  # Empty dict and thus no dl2 path files
```

Comment:
Since `log_from_dl1_dl2` does not have a default value anymore, is this this a reasonable check or should it be 
```python
if not log_from_dl1_dl2:
    ....
```
?

Reply:
I think both are correct. Checking that a dict is empty or that the var does not exists. Won't be able to justify more one or another ^^

Reply:
Yeah, but if you pass an empty list or just `None` to the function now, it wont trigger